Black-Market/Black-Market/Network/Endpoints/SignInEndpoint.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  SessionEndpoint.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 8/2/23.
+//
+
+import Foundation
+
+struct SignInEndpoint: NetworkEndpoint {
+    let params: [String: Any]
+    
+    init(params: [String: Any]) {
+        self.params = params
+    }
+    
+    var absoluteURL: String {
```

Comment:
it is weird to have a URL suffix but the property is a String type.

Reply:
Done